modules/os2forms_organisation_openid_connect/README.md

```diff
@@ -0,0 +1,14 @@
+# OS2Forms Organisation OpenID Connect
+
+Module for OS2Forms Organisation OpenID Connect.
```

Comment:
This is basically just repeating the heading. Tell a bit about what this module actually does.

Reply:
Updated.